server/mongo_data_access.go

```diff
@@ -100,7 +151,10 @@ func (dal *mongoDataAccessLayer) Get(id, resourceType string) (result interface{
 		return nil, convertMongoErr(err)
 	}
 
-	collection := dal.Database.C(models.PluralizeLowerResourceName(resourceType))
+	session := dal.Connection.Copy()
```

Comment:
Maybe a little nitpicky, but by calling it `session`, it could easily be confused for a `mgo.Session`.  Perhaps it should be called `connection` or `conn`?